app/views/application/_header.html.erb

```diff
@@ -11,5 +11,8 @@
   <%= link_to "Blog", '#', class: "header-tab" %>
   <%= link_to "Projects", projects_path, class: "header-tab" %>
   <%= link_to "About", about_path, class: "header-tab" %>
+  <% if current_admin %>
+    <%= link_to "Applications", admin_applications_path, class: "header-tab" %>
+  <% end %>
```

Comment:
Let's eventually move these admin pages into a dropdown like the current website